marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/EntitySearchManager.java

```diff
@@ -79,17 +76,31 @@
 
     public static String QUERY_OPTIONS = "exp-final-entity-options";
     private static Map<String, FacetHandler> facetHandlerMap;
-    private DatabaseClient finalDatabaseClient;
+    private DatabaseClient databaseClient;
```

Comment:
The below variable has a really nice name, which raises the question of - what's this client used for? I think it'd be good to name this "searchDatabaseClient" or something similar to convey what this one is used for.